packages/js/src/Modules/Verto/BaseSession.ts

```diff
@@ -186,17 +186,60 @@ export default abstract class BaseSession {
   }
 
   /**
-   * Attach a listener to the global session level
-   * @return void
+   * Attach an event handler for a specific type of event.
+   *
+   * @param eventName `REQUIRED` Event name.
+   * @param callback `REQUIRED` Function to call when the event comes.
```

Comment:
I don't think we need to add the `required` here as it is in the type of the param.
```suggestion
   * @param eventName `REQUIRED` Event name.
   * @param callback `REQUIRED` Function to call when the event comes.
```